docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -345,7 +345,15 @@ Root permission is required.
 
 :::
 
-1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`.
+1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`. If you choose to use vanilla ffmpeg, instead of jellyfin-ffmpeg, you will need to install the following intel packages.
```

Comment:
@solidsnake1298 We'd be better off linking directly to Intel's Wiki for vanilla ffmpeg users. Because just installing the media-driver package is not enough for QSV. https://github.com/intel/media-driver/wiki

Reply:
I'm assuming I should be removing the apt command block since we are directing users to read Intel's docs.